feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -54,21 +56,40 @@ class HomeViewModel @Inject constructor(
             copy(
                 mapBoxPublicKey = buildConfigProvider.mapBoxPublicKey,
                 mapBoxStyleUri = buildConfigProvider.mapBoxStyleURI,
-                isInitialGeolocationPermissionAsked = getGeolocationPermissionRequestedSettingUseCase()
+                isInitialGeolocationPermissionAsked = getGeolocationPermissionRequestedSettingUseCase(),
+                gpsSettingState = if (gpsSettingsProvider.isGpsSettingsEnabled) {
```

Comment:
Please convert to when

Reply:
is it a good idea using when for boolean? what it looks like?
``` when(bool) {
true-> doA()
false -> doB()
}
```

Reply:
When `if else` is multiline, `when` is more readable

Reply:
done